src/clj/datahike_server/middleware.clj

```diff
@@ -0,0 +1,34 @@
+(ns datahike-server.middleware
+  (:require
+   [datahike-server.config :as dc]
+   [buddy.auth :refer [authenticated?]]
+   [buddy.auth.backends :as buddy-auth-backends]
+   [buddy.auth.middleware :as buddy-auth-middleware]))
+
+(defn auth
+  "Middleware used in routes that require authentication. If request is not
+   authenticated a 401 not authorized response will be returned.
+   Dev mode always authenticates."
+  [handler]
+  (fn [request]
+    (let [dev-mode? (clojure.core/get-in dc/config [:server :dev-mode])]
```

Comment:
Why do you use `clojure.core` namespacing here for `get-in`?

Reply:
I was trying to just use clojure.core because there was a warning that get-in is overwritten somewhere in konserve or something and I thought it is better this way. I'll change it back.